src/test/pmem2_granularity/err2.log.match

```diff
@@ -1 +1 @@
-$(*)requested granularity not available because specified volume is not a direct access (DAX) volume
+$(*)${requested granularity not available because specified volume is not a direct access (DAX) volume}|requested granularity not available because fd doesn't point to DAX-enabled file or kernel doesn't support MAP_SYNC flag (Linux >= 4.15)}
```

Comment:
What about FreeBSD?
With such change this test will pass if we'll print Windows error messages on Linux. I think we need separate test cases for each system.